opensrp-fp/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="108dp"
+    android:height="108dp"
+    android:viewportHeight="108"
+    android:viewportWidth="108">
+    <path
+        android:fillType="evenOdd"
+        android:pathData="M32,64C32,64 38.39,52.99 44.13,50.95C51.37,48.37 70.14,49.57 70.14,49.57L108.26,87.69L108,109.01L75.97,107.97L32,64Z"
+        android:strokeColor="#00000000"
```

Comment:
Move this color & other colors in this file to `colors.xml`

This and other files(launcher drawables) were removed in the previous PR. Contact @zzainulabidin for more and get back on what you two decide